_2020_pages/register.html

```diff
@@ -0,0 +1,36 @@
+---
+layout: 2020_default
+permalink: /register
+title: Registration
+---
+    <!--background color-->
+    <script type="text/javascript">
+        document.write ('<body style="background: Gainsboro; background-attachment: fixed;">')
+    </script>
+
+    <div class="event black-frame">
+        <h1>{{ page.title }}</h1>
+        <h2>Application</h2>
+
+        <p>The deadline is Friday <strong>August 29th 2020</strong> (AoE).</p>
```

Comment:
August 29 is Saturday.
What is "AoE"?

Reply:
Yeah, that should be the Friday then. AoE = Anywhere on earth, or UTC-12

Reply:
Ok. When I google "AoE", I don't get that (mostly it's about Age of Empires)...